_terms/legal-eula.md

```diff
@@ -21,16 +21,16 @@ last_update: 01/10/2016
 on their Service Infrastructures; **Claims** means all warranties, claims, liabilities, suits, losses, damages and expenses, including costs and reasonable attorney’s fees of any kind whether in contract, tort or otherwise; **Customer** means the person so described in the Order Confirmation, or if none, in the Order, who purchases and/or subscribes for the Products; **End-User** means the user authorised by Customer to use the Software and/or Subscription Services; **Intellectual Property Rights** means all rights in any country or jurisdiction in patents, inventions, trade secrets and other rights in know-how, copyrights (including any extensions or renewals), rights affording equivalent protection to copyright, data, rights in database, registered designs, design rights, industrial designs and utility models, trademarks, trade names, trade dress, logos, domain names, business names, and all registrations or applications to register any of the foregoing items: Terms means SixSq’s terms and conditions applying to SixSq’s Products available on SixSq’s website sixsq.com from time to time; **Order** means Customer’s purchase order for Products; **Order Confirmation** means a written confirmation issued by SixSq to Customer accepting Customer’s Order; **Personnel** means a party’s employees, directors, officers agents, contractors and/or representatives; **Services** shall have the meaning ascribed in the Terms; **Service** means SixSq’s online service, for example Nuvla.io, licensed by SixSq to the End User subject to this EULA, including all computer programs and/or software made available to the End User; **Service Infrastructure** means an IT infrastructure, including public and private cloud and edge devices able to run Apps; **Service Offer** means SixSq’s offer issued to Customer for the supply of Products or in response to a tender or request for proposal; **SixSq Materials** means all materials, equipment, documents and other property of SixSq supplied with the Software; **SixSq Partner** means an agent, re-seller, systems integrator or distributor authorised by SixSq to sell, lease and/or provide Products to  Customers; **Software** means the NuvlaBox related edge software used on edge devices to communicate with the Service, including computer programs and/or any firmware (and any modifications or improvements thereto) licensed by SixSq to the End User subject to this EULA, including all computer programs, software and/or any firmware made available to the End User through Subscription Services; **Specification** shall have the meaning ascribed in the Terms; and (**Subscription Services**) means the provision of Services on a subscription basis. Also where the context otherwise requires, words in the singular shall include the plural and in the plural shall include the singular; **Third Party Apps Provider** means third party providers of Apps, including End-Users, for which they have the rights on the software the Apps recipe describes.
```

Comment:
> foregoing items: Terms means SixSq’

is this supposed to be a new term? like "items; **Terms**"

Reply:
Yes. Corrected